Cargo.toml

```diff
@@ -9,7 +9,8 @@ documentation = "https://docs.rs/jobserver"
 description = """
 An implementation of the GNU make jobserver for Rust
 """
-edition = "2018"
+edition = "2021"
+rust-version = "1.66.1"
```

Comment:
nit: I leave the patch version unspecified because it causes unnecessary churn to MSRVs when a dependency bumps the patch

Reply:
Thanks. Update.